### ISSUE TITLE
PMID:28630186 Loss of C-5 Sterol Desaturase Activity Results in Increased Resistance to Azole and Echinocandin Antifungals in a Clinical Isolate of Candida parapsilosis.

### ISSUE THREAD

**Issue by MPiovesana** [PHI-base/curation#156]
Curated by @MPiovesana 
https://canto.phi-base.org/curs/ab37cd93e3786640

**Comment by MPiovesana** [comment]
Ongoing.

**Comment by MPiovesana** [comment]
In this study, authors investigate the azole susceptibility of two clinical isolates of _Candida parapsilosis_, named Isolate 1 (azole sensitive) and Isolate 2 (azole resistant). A point mutation leading to an amino acid substitution in the _ERG3_ gene of Isolate 2 (**ERG3-aaG111R**) is found to be associated with the azole resistance displayed by this isolate. Authors generate deletion mutants of _ERG3_ (_ERG3Δ_) in both Isolate 1 and 2 backgrounds, and also replace the mutant _ERG3_ allele in Isolate 2 with the wild type copy of _ERG3_ from Isolate 1. The effect of the deletion of _ERG3_ in azole susceptibility is also explored in _Candida albicans_ with the generation of deletion mutants in a wild type reference strain (SC5314). 

This is a tricky case as the strains in this study as named Isolate 1 and Isolate 2, and therefore we do not really know to which reference strain of _C. parapsilosis_ they are most similar to. For this reason, I think we can only curate the deletion genotypes below:

_C. parapsilosis_:
**_ERG3Δ_** (Unknown strain): deletion of the wild type copy of _ERG3_ in clinical isolate 1 (Unknown strain selected);


_C. albicans_:
**_ERG3wtΔ_** (SC5314): deletion of the wild type copy of _ERG3_ in reference strain background SC5314.

I thought about creating an allele to represent the replacement of the mutant ERG3 allele of Isolate 2 with the wt allele from Isolate 1, but as both are unknown strains, I am not sure how I could represent this.

AC- I agree with this and have left annotations as they are.

**Comment by MPiovesana** [comment]
**Uniprot IDs:**
_C. parapsilosis_:
A unique reference proteome of _C. parapsilosis_ was identified in UniprotKB (UP000005221), for which a single entry for the ERG3 protein was found (**G8B7T4**).

_C. albicans_:
The reference proteome UP000000559 was chosen for _C. albicans_ as this is identified as a "Standard" proteome (a second one is available but is classed as Outlier (high value)). A single entry for ERG3 was identified (**Q59VG6**).

**Comment by MPiovesana** [comment]
**Genotype creation and annotation:**
Genotypes were created as described above. Genotypes were annotated based on their susceptibility to three azole antifungals (fluconazole, voriconazole, itraconazole). Susceptibility to other chemicals are also tested, but as the main focus of the paper is on azole resistance, these were chosen as representative results.

**Comment by MPiovesana** [comment]
**Question**: a second strain of _C. albicans_ is used by the authors to generate deletion lines of _ERG3_ (strain **BWP17**). The background of this strain is not explained in the text, and I found it rather confusing (Table 4). Thus, I decided not to curate the deletion genotype generated in this background, as I was unsure about how to do it. I don't believe it would be necessary as it shows a similar azole resistance profile to strain SC5314.
